docs/ossm/ossm2-migration/README.md

```diff
@@ -12,7 +12,7 @@ Before you begin to migrate your controlplane from OpenShift Service Mesh 2.6 to
 - Install your 3.0 Openshift Service Mesh Operator. See warning below.
 - Upgrade your Kiali operator to the latest release.
 - Disable the following features on your `ServiceMeshControlPlane`. These fields are unsupported in 3.0 and must be disabled prior to migration.
-  - Network Policy management: `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 `ServiceMeshControlPlane`, you will need to recreate and manage these manually.
+  - Network Policy management: `spec.security.manageNetworkPolicy=false`. If you wish to keep the Network Policies created by the 2.6 `ServiceMeshControlPlane`, you will need to recreate and manage these manually. See [this document](network-policies.md) for more details.
```

Comment:
Could you please also add a link to the cluster-wide doc on following line? https://github.com/openshift-service-mesh/sail-operator/blob/main/docs/ossm/ossm2-migration/cluster-wide/README.md?plain=1#L32

Reply:
https://github.com/openshift-service-mesh/sail-operator/pull/203/commits/19e47dd1d7ea6a8157f1a8d2b92a54909692a205 added a link inside the note